hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HiveHoodieReaderContext.java

```diff
@@ -175,10 +176,14 @@ public ArrayWritable convertAvroRecord(IndexedRecord avroRecord) {
 
   @Override
   public HoodieRecordMerger getRecordMerger(String mergerStrategy) {
-    if (mergerStrategy.equals(DEFAULT_MERGER_STRATEGY_UUID)) {
-      return new HoodieHiveRecordMerger();
+    switch (mergerStrategy) {
+      case DEFAULT_MERGER_STRATEGY_UUID:
+        return new HoodieHiveRecordMerger();
+      case OVERWRITE_MERGER_STRATEGY_UUID:
+        return new OverwriteWithLatestHiveRecordMerger();
+      default:
+        throw new HoodieException(String.format("The merger strategy UUID is not supported, Default: %s, Passed: %s", mergerStrategy, DEFAULT_MERGER_STRATEGY_UUID));
```

Comment:
Should we remove `DEFAULT_MERGER_STRATEGY_UUID` from the error message?  Only logging `"The merger strategy UUID is not supported: " + mergerStrategy`?